source/osh/ui/view/map/CesiumView.js

```diff
@@ -181,37 +181,37 @@ class CesiumView extends View {
    * @param {ImageDraping} styler - The styler allowing the update of the marker
    *
    */
-  updateDrapedImage(styler, timeStamp, options, snapshot) {
+  updateDrapedImage(styler,timeStamp,options,snapshot) {
 
-    var llaPos = styler.platformLocation;
-    var camPos = Cartesian3.fromDegrees(llaPos.x, llaPos.y, llaPos.z);
+    const llaPos = styler.platformLocation;
+    const camPos = Cartesian3.fromDegrees(llaPos.x, llaPos.y, llaPos.z);
 
-    var DTR = Math.PI/180;
-    var attitude = styler.platformOrientation;
-    var gimbal = styler.gimbalOrientation;
+    const DTR = Math.PI/180;
+    const attitude = styler.platformOrientation;
+    const gimbal = styler.gimbalOrientation;
 
     ///////////////////////////////////////////////////////////////////////////////////
     // compute rotation matrix to transform lookrays from camera frame to ECEF frame //
     ///////////////////////////////////////////////////////////////////////////////////
-    var nedTransform = Transforms.northEastDownToFixedFrame(camPos);
-    var camRot = new Matrix3();
+    const nedTransform = Transforms.northEastDownToFixedFrame(camPos);
+    const camRot = new Matrix3();
     Matrix4.getMatrix3(nedTransform, camRot);
-    var rotM = new Matrix3();
+    const rotM = new Matrix3();
 
     // UAV heading, pitch, roll (given in NED frame)
-    var uavHeading = Matrix3.fromRotationZ(attitude.heading * DTR, rotM);
```

Comment:
I don't think it's a good idea to have a 'var' that becomes a 'const' on the same variable here. The intent is not clear. Let's keep it a 'var'.